### ISSUE TITLE
Updating firestore rules does not add historical log to Firestore Console Rules for a non-default database (multiple databases)

### ISSUE THREAD

**Issue by memoricab** [firebase/firebase-tools#7456]
Hello,
We are experiencing an issue which is not reproducible on (default) db but on named db (multiple databases)
I'm not sure if this bug should be owned by firebase-cli since Firestore Console is also involved, I reported directly via form as well. If not related here, we can close.

### [REQUIRED] Environment info

1.13.3

**firebase-tools:**

**Platform:** ubuntu

### [REQUIRED] Test case

firebase deploy --only firestore:named-db --project "my-project"

### [REQUIRED] Steps to reproduce

Create a named db (multiple databases)
Update the rules of named-db as above test case.
Go to Firebase Cloud Firestore Console, select the named-db and  click on Rules tabs.

### [REQUIRED] Expected behavior
I should see the historical log of update on the left side. **-for (default) database expected behaviour is reproducible**

### [REQUIRED] Actual behavior
No new historical log after rules are updated with firebase-cli. **Nevertheless, the rules are updated.**

**Comment by aalej** [comment]
Hey @memoricab, thanks for reporting this! I was able to reproduce the behavior you mentioned. One thing I noticed is that, when using the Firebase CLI the historical tab does not get updated, but when updating the rules via the Firebase Console the historical tab would get updated. In both cases, the rules get updated.

I’ll raise this to our engineering team so that we can verify if this is a bug with the Firebase CLI.

**Comment by memoricab** [comment]
Yes @aalej, forgot to mention sorry, that is correct- if you update the rules of non default db from console it does update the history.

**Comment by Luiz-N** [comment]
2 years later and this bug is still happening... any updates @aalej ?

**Comment by memoricab** [comment]
Hello, this issue has been fixed by firebase support - got confirmation and I've just tested and verified. Thanks.
